src/main/java/org/dependencytrack/tasks/OsvDownloadTask.java

```diff
@@ -39,25 +38,36 @@
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import java.util.Scanner;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
 import static org.dependencytrack.model.ConfigPropertyConstants.VULNERABILITY_SOURCE_GOOGLE_OSV_ENABLED;
 import static org.dependencytrack.model.Severity.getSeverityByLevel;
-import static org.dependencytrack.util.VulnerabilityUtil.*;
+import static org.dependencytrack.util.VulnerabilityUtil.normalizedCvssV3Score;
+import static org.dependencytrack.util.VulnerabilityUtil.normalizedCvssV2Score;
 
 public class OsvDownloadTask implements LoggableSubscriber {
 
     private static final Logger LOGGER = Logger.getLogger(OsvDownloadTask.class);
 
     private static final String OSV_BASE_URL = "https://osv-vulnerabilities.storage.googleapis.com/";
+    public static final String OSV_CONFIG_GROUP = "osv-ecosystems";
     private final boolean isEnabled;
-    private HttpUriRequest request;
+    private final List<String> ecosystems;
+    private static HttpUriRequest request;
```

Comment:
Sharing request objects will lead to race conditions and all other sorts of issues.

Because this field is shared for downloading `ecosystems.txt` and `all.zip`, what will happen if an `all.zip` file is being downloaded while someone requests `/api/v1/ecosystem`?

It's best to scope request objects to where they're used.